packages/inspector/vitest.e2e.config.js

```diff
@@ -13,7 +13,7 @@ export default defineConfig({
     pool: 'forks', // use forks instead of threads for better isolation
     poolOptions: {
       forks: {
-        singleFork: false,
+        singleFork: true, // run tests sequentially in a single fork
```

Comment:
I think I tried this and it made the tests fail inconsistently. But your build seems to be succeeding. Let's keep it and keep an eye on it